src/components/ConfigProvider/ConfigProvider.tsx

```diff
@@ -36,6 +48,30 @@ const ConfigProvider: FC<ConfigProviderProps> = ({
         setFontSize(DEFAULT_FONT_SIZE);
     }, [themeOptions]);
 
+    const handleFocusVisible = useCallback((): void => {
+        if (
+            isFocusVisible &&
+            !focusVisibleElement?.classList.contains(focusVisibleClassName)
+        ) {
+            focusVisibleElement.classList.add(focusVisibleClassName);
+        } else {
+            if (
+                focusVisibleElement?.classList.contains(focusVisibleClassName)
+            ) {
+                focusVisibleElement.classList.remove(focusVisibleClassName);
+                if (focusVisibleElement.classList.length === 0) {
+                    focusVisibleElement.removeAttribute('class');
+                }
+            }
+        }
+    }, [isFocusVisible]);
+
+    useEffect(() => {
+        if (focusVisible) {
+            handleFocusVisible();
+        }
+    }, [handleFocusVisible]);
+
```

Comment:
Can this be a hook itself and be used here

Reply:
This logic is written for the ConfigProvider. Shouldn't it remain here for testability? It relies on the props to function.